pbf/templates/player.html

```diff
@@ -134,7 +134,7 @@ <h4>Selection:</h4>
   <button class="btn" hx-get="{% url 'undo_gain_card' player.id %}" hx-confirm="Are you sure?">Oops! Undo gain card</button>
   {% else %}
   <p>Gain Minor ({{ player.game.minor_deck.all | length }} in deck): <button class="btn" hx-get="{% url 'gain_power' player.id 'minor' '4' %}">4 Cards</button> <button class="btn" hx-get="{% url 'gain_power' player.id 'minor' '6' %}">6 Cards</button> <button class="btn" hx-get="{% url 'take_power' player.id 'minor' %}">1 Card</button>
-  <p>Gain Major ({{ player.game.major_deck.all | length }} in deck): <button class="btn" hx-get="{% url 'gain_power' player.id 'major' '4' %}">4 Cards</button> <button class="btn" hx-get="{% url 'gain_power' player.id 'major' '2' %}">2 Cards</button> <button class="btn" hx-get="{% url 'take_power' player.id 'major' %}">1 Card</button>
+  <p>Gain Major ({{ player.game.major_deck.all | length }} in deck): <button class="btn" hx-get="{% url 'gain_power' player.id 'major' '4' %}">4 Cards</button> {% if player.spirit.name == 'Covets' %} <button class="btn" hx-get="{% url 'gain_power' player.id 'major' '6' %}">6 Cards</button> {% endif %} <button class="btn" hx-get="{% url 'gain_power' player.id 'major' '2' %}">2 Cards</button> <button class="btn" hx-get="{% url 'take_power' player.id 'major' %}">1 Card</button>
```

Comment:
only visible for Covets Gleaming Shards of Earth because no other spirit will ever get access to this ability, as it is part of the hoard